app/models/logs_search.rb

```diff
@@ -22,16 +21,24 @@ def validate_term
     end
   end
 
+  def validate_presence
+    if term.length == 0
```

Comment:
I think `term.length == 0` can be written as `term.length.zero?`. 
I remember getting linting errors for this.

Reply:
Or `term.empty?` (or `term.blank?` if you want to use ActiveSupport)

Reply:
👍